Firestore/Example/Tests/Integration/API/FIRDatabaseTests.m

```diff
@@ -16,6 +16,8 @@
 
 @import FirebaseFirestore;
 
+#import <FirebaseFirestore/FIRCollectionReference.h>
```

Comment:
I'm surprised these are required. `@import FirebaseFirestore;` should have done this for you.

Reply:
Removed. App Code auto-adds them. I need to be more careful here :/